### ISSUE TITLE
TraceTelemetry should also include MessageTemplate as property

### ISSUE THREAD

**Issue by rmandvikar** [serilog/serilog-sinks-applicationinsights#95]
It looks like in commit 1eb4fb4 ("Trace does not include LogLevel and MessageTemplate by default (#36)", 2019-01-28), `includeMessageTemplateAsProperty` was (incorrectly) defaulted to `false` (it was `true` before) as  commit `c2c12d9` ("Exceptions and events should have LogLevel along with other properties (#89)", 2019-02-06) reverts most of commit `1eb4fb4` but does not revert for below method. In package 2.6.0, MessageTemplate is logged as property and in package 2.6.2, it's not. Is this change intentional or a bug?

```
        public static TraceTelemetry ToDefaultTraceTelemetry(
            this LogEvent logEvent,
            IFormatProvider formatProvider,
            bool includeLogLevelAsProperty = false,
            bool includeRenderedMessageAsProperty = false,
            bool includeMessageTemplateAsProperty = false)
        {
            if (logEvent == null) throw new ArgumentNullException(nameof(logEvent));

            var renderedMessage = logEvent.RenderMessage(formatProvider);

            var telemetry = new TraceTelemetry(renderedMessage)
            {
                Timestamp = logEvent.Timestamp,
                SeverityLevel = logEvent.Level.ToSeverityLevel()
            };

            // write logEvent's .Properties to the AI one
            logEvent.ForwardPropertiesToTelemetryProperties(telemetry, formatProvider, includeLogLevelAsProperty, includeRenderedMessageAsProperty, includeMessageTemplateAsProperty);

            return telemetry;
        }
    }
```

**Comment by aloneguid** [comment]
There was quite a bit of activity in this area, but the truth is I just can't remember :) To be honest it's really hard to customise the logic now and I'm putting my thoughts into next major v3 to ease the pain.

**Comment by rmandvikar** [comment]
@aloneguid: Will you be accepting a PR for this fix? I'll see if I can find some time to do this.

**Comment by aloneguid** [comment]
It's already fixed in v3 (3.0.1), but feel free to reopen if it doesn't work.

**Comment by rmandvikar** [comment]
I'd like to submit a PR to fix this bug in 2.6.4. This fix is at patch and should bump to 2.6.5. To what branch should I submit the PR for?

here is the diff: 
https://github.com/serilog/serilog-sinks-applicationinsights/compare/master...rmandvikar:fix-trace-messagetemplate?expand=1

**Comment by rmandvikar** [comment]
@aloneguid If you push a branch at tag `v134`, I can submit a PR for this bug. I can't submit a PR to a tag (see below link) and I can't submit a PR to `master` (see link in my previous message).

https://github.com/serilog/serilog-sinks-applicationinsights/compare/v134...rmandvikar:fix-trace-messagetemplate
